draft-ietf-quic-version-negotiation.md

```diff
@@ -68,6 +68,73 @@ document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
 
+# Server Deployments of QUIC {#server-fleet}
+
+While this document mainly discusses a single QUIC server, it is common for
+deployments of QUIC servers to include a fleet of multiple server instances. We
+therefore define the following terms:
+
+Accepted Versions:
+
+: This is the set of versions supported by a given server instance. More
+specifically, these are the versions that a given server instance will use if a
+client sends a first flight using them.
+
+Negotiated Versions:
+
+: This is the set of versions that a given server instance will send in a
+Version Negotiation packet if it receives a first flight from an unknown
+version. This set will most often be equal to the Accepted Versions set, except
+during short transitions while versions are added or removed (see below).
+
+Fully-Deployed Versions:
+
+: This is the set of QUIC versions that is supported and negotiated by every
+single QUIC server instance in this deployment. If a deployment only contains a
+single server instance, then this set is equal to the Negotiated Versions set,
+except during short transitions while versions are added or removed (see below).
+
+If a deployment contains multiple server instances, it is possible for software
+updates to not happen at the exact same time on all server instances. Because
```

Comment:
```suggestion
updates to happen a few server instances at a time. Because
```
Removes a split infinitive and language that feels excessively informal ("exact same time").